scripts/screeps_tencent_batch_rl_runner.py

```
@@ -2558,6 +2558,36 @@ def validate_positive_policy_update(raw: Any) -> None:
     require_explicit_false_policy_update_safety_flags(raw, "policyUpdate")
     require_explicit_false_policy_update_safety_flags(next_candidate_policy, "policyUpdate.nextCandidatePolicy")
+    validate_positive_policy_update_parameter_change(raw, next_candidate_policy)
+
+
+def validate_positive_policy_update_parameter_change(
+    raw: dict[str, Any],
+    next_candidate_policy: dict[str, Any],
+) -> None:
+    updated_parameters = raw.get("updatedParameters")
+    if not isinstance(updated_parameters, dict) or not updated_parameters:
+        raise BatchRunError("remote policyUpdate.updatedParameters must be a non-empty object when policyUpdateIterations is positive")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)